SCons/Environment.py

```diff
@@ -558,27 +558,46 @@ def s(obj):
 
     subst_target_source = subst
 
-    def backtick(self, command):
+
+    def backtick(self, command) -> str:
+        """Emulate command substitution.
+
+        Provides behavior conceptually like POSIX Shell notation
+        for running a command in backquotes (backticks) by running
+        ``command`` and returning the resulting output string.
+
+        This is not really a public API any longer, it is provided for the
```

Comment:
It's not documented, and you can't get to it - though presumably if you tried to call it by name it would work. It's not in the two sections marked by big comments about being public...
```py
    #######################################################################
    # Public methods for manipulating an Environment.  These begin with
...
    #######################################################################
    # Public methods for doing real "SCons stuff" (manipulating
```